lib/hammer_cli_foreman/audit.rb

```diff
@@ -61,6 +61,7 @@ def extend_data(audit)
           field :old, _("Old"), Fields::LongText, :hide_blank => true
           field :new, _("New"), Fields::LongText, :hide_blank => true
         end
+        field :request_uuid, _("Request UUID")
       end
 
```

Comment:
Right away after the changes in previous comment, we can use
```ruby
extend_output_definition do |definition|
  definition.insert(:replace, :request_uuid) do
    field :request_uuid, _("Request UUID"), Fields::Field, sets: ['ALL', 'DEFAULT']
  end
end
```
here, so the `Request UUID` field is shown by default.